packages/path-provider/android/src/main/java/io/flutter/plugins/path_provider/PathProviderPlugin.java

```diff
@@ -5,13 +5,16 @@
 package io.flutter.plugins.path_provider;
 
 import android.app.Activity;
+import android.os.Environment;
 import io.flutter.plugin.common.MethodCall;
 import io.flutter.plugin.common.MethodChannel;
 import io.flutter.plugin.common.MethodChannel.MethodCallHandler;
 import io.flutter.plugin.common.MethodChannel.Result;
 import io.flutter.plugin.common.PluginRegistry.Registrar;
 import io.flutter.util.PathUtils;
 
+
+
```

Comment:
I have no write access to the github repository, so I can not actually do the merge myself. I have squashed all the commits on my fork, so it can be merged at any point.